packages/web/src/pages/sign-up-page/pages/SelectGenrePage.tsx

```diff
@@ -52,40 +53,76 @@ export const SelectGenrePage = () => {
     [dispatch, navigate]
   )
 
+  const { isMobile } = useMedia()
+
   return (
-    <div>
-      <div>
-        <p>{displayName}</p>
-        <p>{handle}</p>
-      </div>
-      <h1>{messages.header}</h1>
-      <p>{messages.description}</p>
+    <Flex direction='column' gap={isMobile ? '2xl' : '4xl'}>
+      <AccountHeader />
+      <Flex direction='column' gap={isMobile ? 'xl' : '2xl'}>
+        <Flex direction='column' gap={isMobile ? 's' : 'l'} ph='l'>
+          <Text
+            variant='heading'
+            size={isMobile ? 'm' : 'l'}
+            color='heading'
+            css={{ textAlign: isMobile ? 'left' : 'center' }}
+          >
+            {messages.header}
+          </Text>
+          <Text
+            variant='body'
+            size={isMobile ? 'm' : 'l'}
+            css={{ textAlign: isMobile ? 'left' : 'center' }}
+          >
+            {messages.description}
+          </Text>
+        </Flex>
 
-      <Formik initialValues={initialValues} onSubmit={handleSubmit}>
-        {({ values, setValues }) => {
-          const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
-            setValues({ ...values, [e.target.name]: !values[e.target.name] })
-          }
+        <Formik initialValues={initialValues} onSubmit={handleSubmit}>
+          {({ values, setValues }) => {
+            const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+              setValues({ ...values, [e.target.name]: !values[e.target.name] })
+            }
 
-          return (
-            <Form>
-              {genres.map((genre) => {
-                const { label, value } = genre
-                return (
-                  <GenrePill
-                    key={value}
-                    label={label}
-                    name={value}
-                    checked={!!values[value]}
-                    onChange={handleChange}
-                  />
-                )
-              })}
-              <Button type='submit'>{messages.continue}</Button>
-            </Form>
-          )
-        }}
-      </Formik>
-    </div>
+            return (
+              <Flex
+                as={Form}
+                alignItems={isMobile ? 'flex-start' : 'center'}
+                direction='column'
+              >
+                <Flex
+                  css={{ maxWidth: 608 }}
+                  justifyContent={isMobile ? 'flex-start' : 'center'}
+                  alignItems='flex-start'
+                  gap='s'
+                  wrap='wrap'
+                  mb='4xl'
+                  ph='l'
+                >
+                  {genres.map((genre) => {
+                    const { label, value } = genre
+                    return (
+                      <SelectablePill
+                        key={value}
+                        size='large'
+                        type='checkbox'
```

Comment:
🔥

Reply:
@amendelsohn check it out now, updated to be even better, uses the built-in formik checkbox field now